core/src/test/scala/org/apache/spark/sql/delta/ShowTableColumnsSuite.scala

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.delta
+
+import org.apache.spark.sql.delta.test.DeltaSQLCommandTest
+import org.apache.spark.sql.functions.struct
+import org.apache.spark.sql.{AnalysisException, DataFrame, QueryTest, Row}
+import org.apache.spark.sql.test.SharedSparkSession
+import org.apache.spark.util.Utils
+
+import java.io.File
+
+class ShowTableColumnsSuite
+    extends QueryTest
+    with SharedSparkSession
+    with DeltaSQLCommandTest
+    with DeltaTestUtilsForTempViews {
+
+  import testImplicits._
+
+  private val outputColumnNames = Seq("col_name")
+  private val outputColumnValues = Seq(Seq("column1"), Seq("column2"))
+
+  protected def checkResult(
+      result: DataFrame,
+      expected: Seq[Seq[Any]],
+      columns: Seq[String]): Unit = {
```

Comment:
Looks like `columns` is always `outputColumnNames`. Can we have at least one test to verify the output schema? Just to make sure we have only one output column `col_name`.

Reply:
Added a column name check at https://github.com/delta-io/delta/pull/1203/files#diff-c45cbb883d008982239860ff3b2847ef46b957768edd1fc026f046fb62fe06bcR45